UI/frontend-plugins/frontend-tools/data/scripts/instant-replay.lua

```diff
@@ -22,27 +25,45 @@ function try_play()
 
 	obs.obs_output_release(replay_buffer)
 
+	if path == last_replay then
+		path = nil
+	end
+
 	-- If the path is valid and the source exists, update it with the
 	-- replay file to play back the replay.  Otherwise, stop attempting to
 	-- replay after 10 seconds
```

Comment:
You would need to update this comment to reflect that it stops attempting after `max_attempts` rather than after 10 seconds.